otx/algorithms/common/adapters/mmcv/utils/config_utils.py

```diff
@@ -161,7 +161,93 @@ def fromfile(filename, use_predefined_variables=True, import_custom_modules=True
         cfg_dict, cfg_text = MPAConfig._file2dict(filename, use_predefined_variables)
         if import_custom_modules and cfg_dict.get("custom_imports", None):
             import_modules_from_strings(**cfg_dict["custom_imports"])
-        return Config(cfg_dict, cfg_text=cfg_text, filename=filename)
+        return MPAConfig(cfg_dict, cfg_text=cfg_text, filename=filename)
+
+    @property
+    def pretty_text(self):
+        """Make python file human-readable."""
```

Comment:
If this function comes from the MMConfig, how about writing where it comes from?
```python
"""Make python file human-readable.""" --> """Modified version of MMConfig ~~~"""
```

Reply:
That's a good idea. I'll update it.